projects/objects/geometries/protos/Rectangle.proto

```diff
@@ -28,10 +28,10 @@ PROTO Rectangle [
   IndexedFaceSet {
     coord Coordinate {
       point [
-        %<= s0 >% %<= t0 >% 0
-        %<= s1 >% %<= t0 >% 0
-        %<= s1 >% %<= t1 >% 0
-        %<= s0 >% %<= t1 >% 0
+        %<= s0 >% 0 %<= t0 >%
+        %<= s1 >% 0 %<= t0 >%
+        %<= s1 >% 0 %<= t1 >%
+        %<= s0 >% 0 %<= t1 >%
```

Comment:
Somehow, I think it was fine before. For example, the floor is also plane and the z-axis is orthogonal to the plane.